.gitignore

```diff
@@ -1 +1,2 @@
 target/
+.idea/
```

Comment:
Please don't put IDE specific stuff in the project .gitignore (put it in your global instead). I know that the Scala.js project itself differs on that for historic reasons. For new projects we'd stick with that convention.